application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -11,17 +10,13 @@
 import net.dv8tion.jda.api.requests.restaction.MessageCreateAction;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
 import org.togetherjava.tjbot.db.Database;
-import org.togetherjava.tjbot.db.generated.tables.records.PendingRemindersRecord;
 import org.togetherjava.tjbot.features.Routine;
 
 import javax.annotation.Nullable;
-
-import java.awt.Color;
+import java.awt.*;
```

Comment:
star-import on `java.awt` is "forbidden", since it quickly leads to bugs with `java.awt.List` vs `java.util.List`. please configure ur IDE import-settings to disallow star import on java.awt, thanks

Reply:
fixed it for this one, not sure how to change import settings but ill look into it

Reply:
`intellij settings > code style > imports > dont use * for ...` sth like that